changelog.d/20241128_114826_PL-133180-agent-fix-cold-reboot-merge_scriv.md

```diff
@@ -0,0 +1,19 @@
+<!--
+
+A new changelog entry.
+
+Delete placeholder items that do not apply. Empty sections will be removed
+automatically during release.
+
+Leave the XX.XX as is: this is a placeholder and will be automatically filled
+correctly during the release and helps when backporting over multiple platform
+branches.
+
+-->
+
+### Impact
+
+
+### NixOS 24.05 platform
```

Comment:
@dpausp: I've missed this earlier, but discovered it during the release process:

> Leave the XX.XX as is: this is a placeholder

Please adhere to this the next time.